lua/arcphone/client/aa_phoneos_core.lua

```diff
@@ -455,7 +455,9 @@ function ARCPhone.PhoneSys:Init_Final()
 	self:SetLoading(-2)
 	self:OpenApp("home")
 	self.Booted = true
-	self:AddMsgBox("BETA VERSION","This is an unfinished BETA version of ARCPhone.\nMost of the core functionality is here, and it works, but there's still a bit more to go. Because of this, features might be added, changed, replaced, or even removed.\nThank you for supporting ARitz Cracker! :)\n(Press ENTER to close this window)","round-info-button")
+	if not ARCPhone.Settings.disable_beta_message then
+		self:AddMsgBox("BETA VERSION","This is an unfinished BETA version of ARCPhone.\nMost of the core functionality is here, and it works, but there's still a bit more to go. Because of this, features might be added, changed, replaced, or even removed.\nThank you for supporting ARitz Cracker! :)\n(Press ENTER to close this window)","round-info-button")
+	end
 	--self:AddMsgBox("My excuse for a tutorial","Use the Arrow keys to move the cursor. Press BACKSPACE to go back, press CTRL to access the context menu (It's kinda like right-clicking), and press ENTER to select.","info")

```

Comment:
Good addition, especially since you're not really supporting _me_ anymore

Reply:
Sorry if i did something wrong :eyes:
i didnt thought about that a "this is a beta" message reduce the support of the original creator in any way

but in my opinion something like this should be an option to disable for the server configurator

but if you dont agree on this i can revert this commit